src/pages/ProductDetails/ProductDetails.tsx

```diff
@@ -1,5 +1,10 @@
+import { Playwood } from './Components/Playwood/Playwood';
 import './ProductDetails.scss';
 
-export const ProductDeatils = () => {
-  return <></>;
-};
+export const ProductDetails =()=>{
+    return(
+        <>
+        <Playwood />
```

Comment:
and what you gonna do if you will need to display another product?)
you will create one more component?)